index.Rmd

```diff
@@ -191,7 +189,7 @@ plot(gewataSub, 1)
 You should see on the resulting plot that the original image has been cropped.
 
 ## Creating layer stacks
-To end this section on general files and raster object manipulations, we will see how multi-layer objects can be created from multiple single-layer objects. Let's create a multi-layer NDVI object, composed of NDVI layers derived from Landsat acquisitions at different dates. We first need to fetch the data file by file.
+To end this section on general files and raster object manipulations, we will see how multi-layer objects can be created from multiple single-layer objects. We have already derived the [Normalized Difference Vegetation Index (NDVI)](https://gisgeography.com/ndvi-normalized-difference-vegetation-index/) from the Landsat 7 imagery in advance for you. The resulting product is an image (2-dimensional SpatRaster) where high pixel values represent green vegetated areas. The dataset that we are working with is a series of NDVI images of a region in Tahiti over time. Let's create a multi-layer NDVI object, composed of NDVI layers derived from Landsat acquisitions at different dates. We first need to fetch the data file by file.
```

Comment:
Many students may not be knowledgeable on NDVI. I therefore added a few lines to walk them through NDVI and general satellite imagery, with some helpful links.